app.js

```diff
@@ -68,9 +66,9 @@ function insertNextImg() {
   if (imageWrapper.firstElementChild.tagName === 'P') {
     insertedImg.src = 'img/s0.jpg';
   } else {
-    let currentImgSrc = imageWrapper.firstElementChild.src
-    let currentImgNumberIndex = currentImgSrc.indexOf('.jpg') - 1;
-    let nextImgNumber = Number(currentImgSrc[currentImgNumberIndex]) + 1;
+    const currentImgSrc = imageWrapper.firstElementChild.src
+    const currentImgNumberIndex = currentImgSrc.indexOf('.jpg') - 1;
+    const nextImgNumber = Number(currentImgSrc[currentImgNumberIndex]) + 1;
```

Comment:
you could use string template literal one line below:
```
insertedImg.src = `img/s${nextImgNumber}.jpg`
```

i have a task for you, but it doesn't have to be done in this pr. It's up to you.

Let's make an object called `state` -> it will be at the beggining of this file. For first step, it can keep currentNumberIndex, by default set to 0. 
```
const state = {
 currentNumberIndex: 0,
};
```
When click, you can increment this value and use it to define the picture.
```
insertedImg.src = `img/s${state.currentNumberIndex}.jpg`
```
Your current aproach is opposite to what we should think about application and programming. What I mean, at this moment you base on this what you are displaying to the user (checking the currentImgSrc, taking the index from there, etc). 
Try to think opposite -> at first you have a date and then you are displaying user interface base on this data. 

In the software engineering we try to call state the object containing dynamic data depending on what user is doing in the app. In the future, you could try to move there everything.